Corona-Warn-App/src/main/res/values/strings.xml

```diff
@@ -1373,8 +1347,6 @@
     <string name="submission_status_card_positive_result_contact">"The public health authority may contact you within the next few days."</string>
     <!-- YTXT: text for share result card-->
     <string name="submission_status_card_positive_result_share">"Share your random IDs so that others can be warned."</string>
-    <!-- YTXT: text for share result card-->
-    <string name="submission_status_card_positive_result_illness">"Falls Sie Risikofaktoren für eine schwere COVID-19-Erkrankung haben, lassen Sie sich ärztlich beraten. Bekannte Risikofaktoren sind z.B. Alter über 60 Jahre, Immunschwäche, Diabetes und hohes Übergewicht."</string>
```

Comment:
why `submission_status_card_positive_result_illness` has been deleted? 
We added this key to `values-de/strings.xml` to be translated.